src/seedu/addressbook/data/person/Tagging.java

```diff
@@ -0,0 +1,32 @@
+package seedu.addressbook.data.person;
+
+import seedu.addressbook.data.person.Person;
+
+public class Tagging {
```

Comment:
Missing header comment. All non trivial methods should have javadoc format header comments.